server/src/controllers/GameControllers/gameConstants.js

```diff
@@ -0,0 +1,13 @@
+const SNAKE_CONSTANTS = {
```

Comment:
Maybe make this configurable using a `.env`?

Reply:
Could it belong in .env file? it is not related to the environment. @niinpatel thoughts?

Reply:
I missed this! Nope, I don't think this belongs to .env